server/Dockerfile

```
@@ -1,3 +1,9 @@
-FROM ghcr.io/deephaven/server:${VERSION:-latest}
-# pip install any of the plugins required on the server
-RUN pip install deephaven-plugin-matplotlib
+# syntax=docker/dockerfile:1.4
+
+FROM ghcr.io/deephaven/web-plugin-packager:latest as build
+RUN ./pack-plugins.sh @deephaven/js-plugin-matplotlib
+
+FROM ghcr.io/deephaven/server:latest
+RUN pip install --no-cache-dir deephaven-plugin-matplotlib
```

```suggestion
RUN pip install --no-cache-dir deephaven-plugin-matplotlib matplotlib
```

Matplotlib should be installed in the Dockerfile as well

---

@mofojed doesn't `deephaven-plugin-matplotlib` depend on `matplotlib`? We shouldn't need to re specify dependency IMO.

---

That said, it's _not_ bad to be explicit here since we *do* rely directly on matplotlib.